panel/src/components/Forms/Blocks/Block.vue

```diff
@@ -264,6 +264,7 @@ export default {
 					submit: this.submit
 				},
 				props: {
+					expandable: true,
```

Comment:
Since this isn't a dynamic value but always the same, it would be better to not pass this in the call but rather declare in `BlockDrawer` the `expandable` prop's default as `true`.